CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 CHANGELOG
 =========
 
+## 0.14.0
+
+- Support `CSI ? 5 W` to reset tabs stops to 8 columns
```

Comment:
I'll be able to give a proper review later today, but this changelog message will certainly have to change. It's not guaranteed that tab stops are going to be 8 columns wide. Should say something like "... to the default" or similar.

Reply:
Note that this escape sequences is [`DECST8C`](https://vt100.net/docs/vt510-rm/DECST8C.html), which is supposed to "set a tab stop at every eight columns starting with column 9" (quoting the VT510 and VT520/VT525 programmer manuals).  If that's not what it's doing, I'd think that's a problem with the implementation rather than the change log.